src/transform/src/join_implementation.rs

```diff
@@ -609,27 +627,27 @@ mod delta_queries {
                             key.iter().map(|k| format!("{k}")).join(" = ")
                         ))
                         .join(" "),
-                    "insufficient arrangements",
+                    "insufficient arrangements for delta join",
                 );
 
                 // Differential joins need to arrange every intermediate result: #inputs - 1 such arrangements.
                 // Delta joins only arrange inputs... if we would have fewer net arrangements, delta will be a better deal.
                 if insufficiently_arranged_orders > expected_differential_arrangements {
                     return Err(TransformError::Internal(String::from(
-                        "delta plan not viable",
+                        "delta join not viable",
                     )));
                 } else if !eager_delta_joins {
                     return Err(TransformError::Internal(String::from(
-                        "delta plan viable but not used (eager delta joins disabled)",
+                        "delta join viable but not used (eager delta joins disabled)",
                     )));
                 }
-            }
 
-            tracing::info!(
-                new_delta_input_arrangements = insufficiently_arranged_orders,
-                expected_differential_arrangements = expected_differential_arrangements,
-                "using a delta join instead of a differential join"
-            );
+                tracing::info!(
```

Comment:
Change this to

```rust
tracing::debug!(
    new_delta_input_arrangements = insufficiently_arranged_orders,
    expected_differential_arrangements = expected_differential_arrangements,
    "eager delta join instead of a differential join"
);
```

Then I believe setting the `log_filter` system var to
```
mz_transform::join_implementation=DEBUG,INFO
```
either in your staging environment (via LaunchDarkly) or with `ALTER SYSTEM SET log_filter = $value` (if you are running locally) would let you see these comments on your environment without emitting this line everywhere else.

Reply:
Will do! (Thank you for the tip on setting `log_filter`, one of several things I cannot keep in my head 😢.)